client/src/components/QandA/QuestionsList.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import config from '../../../../config.js';
+import QuestionCard from './QuestionsList/QuestionCard.jsx';
+import AddQuestionOrAnswer from './QuestionsList/QuestionCard/AddQuestionOrAnswer.jsx';
+
+const host = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp';
+const headers = {
+  'Authorization': `${config.TOKEN}`
```

Comment:
the way you are pulling in your token here will turn into a merge conflict because not everyone has stored their tokens in the same fashion. Kavi and i are storing it as a const in config.js then importing it like so:
import {token} from 'config.js'

Reply:
Resolved. Importing thru destructuring in QuestionsList.jsx. Also, added base url to config.js.